packages/msgpack_c.rb

```diff
@@ -1,50 +1,38 @@
 # Adapted from Arch Linux msgpack-c PKGBUILD at:
 # https://github.com/archlinux/svntogit-community/raw/packages/msgpack-c/trunk/PKGBUILD
 
-require 'package'
+require 'buildsystems/cmake'
 
-class Msgpack_c < Package
+class Msgpack_c < CMake
   description 'An efficient object serialization library'
   homepage 'https://msgpack.org/'
-  version '4.0.0'
+  version '6.0.0'
   license 'custom:Boost'
   compatibility 'all'
-  source_url 'https://github.com/msgpack/msgpack-c/releases/download/c-4.0.0/msgpack-c-4.0.0.tar.gz'
-  source_sha256 '420fe35e7572f2a168d17e660ef981a589c9cbe77faa25eb34a520e1fcc032c8'
+  source_url 'https://github.com/msgpack/msgpack-c/releases/download/c-6.0.0/msgpack-c-6.0.0.tar.gz'
+  source_sha256 '3654f5e2c652dc52e0a993e270bb57d5702b262703f03771c152bba51602aeba'
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '6f135003a98c4c33e936257a416166187753d66618c8663176e60950314bbffd',
-     armv7l: '6f135003a98c4c33e936257a416166187753d66618c8663176e60950314bbffd',
-       i686: '54e7a62e5a889d34046131de43601a41762ce54fdc90751043e4b8bb4a3e8fdf',
-     x86_64: 'c4ce44f936fa4f393446b3193aca6cf911d23fc95c409a5d02ba500afe36706c'
+    aarch64: '954cc5887ce7be419b0571bbfd8da72c436a83d0d7c0ac4f0033e8191cc69be1',
+     armv7l: '954cc5887ce7be419b0571bbfd8da72c436a83d0d7c0ac4f0033e8191cc69be1',
+       i686: '23f44da7a26b62bc6b38be11b7c2a5cefc1b2ac11a5d87982f73a4a2edb8ad03',
+     x86_64: '4171f5ad1d9e9b4e40b06e0e9f68173edace4f25d7d95e8b1f9e581e45a87f7d'
   })
 
-  depends_on 'cmake' => :build
   depends_on 'doxygen' => :build
-  depends_on 'graphviz' => :build
+  depends_on 'glibc' # R
 
   def self.patch
```

Comment:
Send these improvements upstream? (i.e. add support for passing `-DCMAKE_INSTALL_PREFIX` the standard way)

Reply:
Feel free!